src/features/alcohol/AlcoholPreview/AlcoholPreview.tsx

```diff
@@ -0,0 +1,36 @@
+import classNames from 'classnames/bind';
+import Image from 'next/image';
+
+import styles from './AlcoholPreview.module.scss';
+
+const cx = classNames.bind(styles);
+
+type AlcoholPreviewProps = {
+  name: string;
+  brand: string;
+  description: string;
+  imgSrc: string;
+};
+
+const AlcoholPreview = (props: AlcoholPreviewProps) => {
```

Comment:
이런식으로 통일하면 좋을 것같아용
(이유: default 값 설정하기가 편하다)
```suggestion
const AlcoholPreview = ({ name, brand, }: AlcoholPreviewProps) => {
```

Reply:
반영했습니다!